### PR TITLE
feat: core-216 updating icon in FavoriteStar.vue replaced with new bo…

### PR DESCRIPTION
**Pull request by BoulderBrains**
…okmark icon

[Core-216](https://kiva.atlassian.net/browse/CORE-216)

- I've removed the old icon from the FavoriteStar.vue component and replaced it with a new Material Design icon.
- I've translated a few of the css declarations into tailwind classes and cleaned that up in LoanCardImage.vue
- Added a note in FavoriteStar.vue about not using that component moving forward. 

**Finished product:** 
Screen Shot 2022-01-12 at 1 39 03 PM

### CODE REVIEW

File: src/components/LoanCards/FavoriteStar.vue [hunk 1]
```diff
@@ -1,18 +1,50 @@
...
+	*
+	*
+	*
+	*
+	*-->
+	<span>
+		<button
+			v-if="!isFavorite"
+			class="tw-inline-flex tw-p-1 tw-cursor-pointer tw-bg-primary tw-rounded tw-m-0.5"
+			@click="toggleFavorite()"
```
Comment (mcstover) [inline src/components/LoanCards/FavoriteStar.vue:21]: Please bring the kv-track events back to these buttons

Comment (BoulderBrains) [reply src/components/LoanCards/FavoriteStar.vue:21]: Thank you for catching that.

File: src/components/LoanCards/FavoriteStar.vue [hunk 2]
```diff
@@ -1,18 +1,52 @@
...
+			@click.prevent.stop="toggleFavorite()"
+			v-kv-track-event="['Lending', 'click-Favorite star', 'Favorite', loanId, loanId]"
+		>
+			<kv-material-icon
+				class="tw-text-action tw-w-3"
+				:icon="mdiBookmarkOutline"
+			/>
+		</button>
+		<button
+			v-if="isFavorite"
```
Comment (mcstover) [inline src/components/LoanCards/FavoriteStar.vue:30]: I think you can use the `isFavorite` value to simplify this template to only a single button instance.
- remove v-if and use a ternary on the icon to switch between the icon type.
- also use a ternary in your analytics even to swap between `Favorite` and `Un-Favorite`.

File: src/components/LoanCards/FavoriteStar.vue [hunk 3]
```diff
@@ -1,18 +1,52 @@
...
+			class="tw-inline-flex tw-p-1 tw-cursor-pointer tw-bg-primary tw-rounded tw-m-0.5"
+			@click.prevent.stop="toggleFavorite()"
+			v-kv-track-event="['Lending', 'click-Favorite star', 'Favorite', loanId, loanId]"
```
Comment (mcstover) [inline src/components/LoanCards/FavoriteStar.vue:33]: Here for the 'Favorite' field.

File: src/components/LoanCards/FavoriteStar.vue [hunk 4]
```diff
@@ -1,18 +1,52 @@
...
+		>
+			<kv-material-icon
+				class="tw-text-action tw-w-3"
+				:icon="mdiBookmark"
+			/>
+		</button>
+	</span>
```
Comment (mcstover) [inline src/components/LoanCards/FavoriteStar.vue:40]: here to swap the icon.

**Review by mcstover — APPROVED** [review]
Thank you, Nice Cleanup!

**Review by ryan-ludwig — APPROVED** [review]

**Pull request merged**